tests/drivers/counter/counter_basic_api/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,77 @@
+&timer120 {
+	status = "okay";
+	prescaler = <7>;
+};
```

Comment:
These nodes should most probably be sorted, meaning that `timer120` should be after `timer022`.